datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -46,20 +46,20 @@ mmh3==4.0.1; python_version > '3.0'
 oauthlib==3.1.0; python_version < '3.0'
 oauthlib==3.2.2; python_version > '3.0'
 openstacksdk==1.5.0; python_version > '3.0'
-oracledb==1.3.2; python_version >= '3.8'
-orjson==3.9.7; python_version > '3.0'
-packaging==23.1; python_version > '3.0'
+oracledb==1.4.2; python_version >= '3.8'
```

Comment:
Could you revert this one? I'll add it to the exclusion list, sorry I forgot that 

Context https://datadoghq.atlassian.net/browse/AI-3442

Reply:
FTR https://github.com/DataDog/integrations-core/pull/16155

Reply:
Done, thanks for the catch